index.html

```diff
@@ -7,28 +7,58 @@
     <meta name="apple-mobile-web-app-capable" content="yes">
   </head>
   <body>
-    <ul>
-      <li><a href="BookReaderDemo/demo-simple.html">Simple</a></li>
-      <li><a href="BookReaderDemo/view_mode.html">Set initial view mode</a></li>
-      <li><a href="BookReaderDemo/toggle_controls.html">Toggle and override individual controls</a></li>
-      <li><a href="BookReaderDemo/demo-fullscreen.html">Fullscreen with mobile nav</a></li>
-      <li><a href="BookReaderDemo/demo-fullscreen-mobile.html">Inline on page, with mobile nav on fullscreen</a></li>
-      <li><a href="BookReaderDemo/demo-vendor-fullscreen.html">Vendor native fullscreen</a></li>
-      <li><a href="BookReaderDemo/demo-advanced.html">Advanced</a></li>
-      <li><a href="BookReaderDemo/demo-preview-pages.html">Preview Pages</a></li>
-      <li><a href="BookReaderDemo/demo-embed.html">Embed</a></li>
-      <li><a href="BookReaderDemo/demo-multiple.html">Multiple on same page</a></li>
-      <!-- plugin.search.js -->
-      <li><a href="BookReaderDemo/demo-internetarchive.html?ocaid=theworksofplato01platiala">From Internet Archive</a></li>
-      <!-- plugin.search.js -->
-      <li><a href="BookReaderDemo/demo-internetarchive.html?ocaid=adventureofsherl0000unse">From Internet Archive - a book with CHAPTERS</a></li>
-      <li><a href="BookReaderDemo/demo-iiif.html">IIIF</a></li>
-      <li><a href="BookReaderDemo/demo-autoplay.html">Autoplay (kiosk mode)</a></li>
-      <li><a href="BookReaderDemo/demo-plugin-menu-toggle.html">Plugin: Full screen menu toggle</a></li>
-      <li><a href="BookReaderDemo/immersion-mode.html">Start in immersion (fullscreen) mode</a></li>
-      <!-- plugin.search.js -->
-      <li><a href="BookReaderDemo/immersion-1up.html">Start in immersion mode on 1up default item</a></li>
-      <li><a href="BookReaderDemo/viewmode-cycle.html">Replace view mode buttons with view mode cycler</a></li>
-     </ul>
```

Comment:
why have you removed these demo links from here. I mean it help us alot to navigate on specific feature to debug or implement new things on.

Reply:
Oh, you said, you won't merge it.